index.js

```diff
@@ -10,56 +9,37 @@ client.on("ready", () => {
   console.log("Bot funcionando 🤖");
 });
 
-client.on("messageCreate", (msg) => {
-  if (msg.content.startsWith(prefixRedaction)) {
-    let args = msg.content.split(" ");
-    let argsSliced = args.slice(1).join(" ");
-    if (argsSliced.length === 0) {
-      msg.reply({
-        content: `Ops! Parece que você esqueceu de inserir argumentos 😬`,
-      });
-      if (argsSliced.length === 1) {
-        msg.reply({
-          content: `Ops! Parece que você esqueceu de inserir o nome do artigo 😬`,
-        });
-      }
-    } else {
-      msg.reply({
-        content: `${msg.author} acabou  de mandar o artigo "**${argsSliced}**" para a fila de revisão 🚀 \n\n Divirta-se, <@257316997707071491> 😁`,
-      });
-    }
+client.on("messageCreate", (message) => {
+  // Ignores bot messages
+  if (message.author.bot) return;
+
+  // Ignores commandName and extract arguments
+  const args = message.content.split(' ').slice(1);
+
+  if (args.length === 0) {
+    return message.reply({ content: `Ops! Parece que você esqueceu de inserir argumentos 😬` });
+  }
+
+  if (message.content.startsWith(prefixRedaction)) {
+    const postName = args.join(' '); // commandName postName
```

Comment:
If you are going to keep the comments, remove the commandName here, because args.join(' ') refers only to the postName

Reply:
I decided to remove all extra comments. [commit#b8727cf](https://github.com/Lavrudin/diolinux-team-bot/pull/1/commits/b8727cf73001b8c0096724ff1b5ca83b4adbbc6c)